### ISSUE TITLE
Theme changing doesn't work in Firefox/Gecko

### ISSUE THREAD

**Issue by dustojnikhummer** [giterlizzi/dokuwiki-template-bootstrap3#582]
### Description

Theme selector in Navbar doesn't work in Firefox

### Steps to reproduce

1. Go to https://laptopwiki.eu in Firefox (desktop, mobile)
2. In the top right select the Color Palete icon
3. Change theme (default is Darkly)

**Expected behavior:** Theme changes

**Actual behavior:** Theme stays at Darkly even if parameter <code> ?bootswatch-theme=slate </code> is passed

Same thing happens on Firefox on Android, Windows, Linux (Debian)
Works perfectly fine in Chromium based browser (can't confirm Safari as I don't own any Apple devices). 
TLDR: Seems to be a Gecko issue

### Versions

* [Bootstrap3 Template] 2022-07-27
* [DokuWiki]  2022-07-31a "Igor"
* [Plugins]
* [PHP] 7.4.33
* [Browser]  Firefox 108.0.2 (64-bit) - Windows and Debian

### Screenshots or Logs

Template settings

https://i.imgur.com/W0MBVdl.png

**Comment by giterlizzi** [comment]
FIXED in development branch

nice website :)

**Comment by dustojnikhummer** [comment]
> FIXED in development branch
> 
> nice website :)

Works perfectly after replacing the theme-switcher.php file
Very much thanks, now I can move to Firefox!
